tools/cmake/modules/cifuzz-functions.cmake

```
@@ -49,6 +49,8 @@ function(enable_fuzz_testing)
           # Undefine NDEBUG, which is explicitly defined by the RelWithDebInfo CMake configuration, so that asserts are
           # kept.
           -UNDEBUG
+          # Do optimizations which don't harm debugging
```

```suggestion
          # Limit to optimizations which don't harm debugging
```
At least that's the intention since `-O2` might still appear on the command line before this flag. We have to confirm that it actually overrides that flag in the expected way.